catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/ReliableResourceDownloaderConfig.java

```diff
@@ -43,6 +47,10 @@
 
     private DownloadsStatusEventListener eventListener;
 
+    private DownloadStatusInfo downloadStatusInfo;
```

Comment:
Not sure if we should put things like `DownloadStatusInfo` and `ExecutorService` in this class. Since it is called `Config`, should it only contain things configured by the administrator through the admin console and not object dependencies?